1_beginner/chapter5/practice/alternating.py

```diff
@@ -1,12 +1,12 @@
-""" 
+"""
+Alternating
```

Comment:
@code-for-tomorrow/python who submitted this practice problem? I'm wondering because I noticed there's no solution for it...